### ISSUE TITLE
Ctrl-modified cursor keys no longer output any keycode since 2.7.5

### ISSUE THREAD

**Issue by Camusensei** [mintty/mintty#646]
Hello, using mintty 2.7.5 in Cygwin, Ctrl+Arrow no longer maps to anything outside of applications.
Examples:
- In bash, Ctrl+Arrow has no effect
- Inside an app like vim, Ctrl+Arrow behaves normally.

This is a regression as mintty 2.7.4 works correctly.

**Comment by Camusensei** [comment]
I meant specifically Cursor left/right but it's exactly the same with up and down.
Okay, I installed back version 2.7.5 to test `Ctrl+V` (I previously only tested with `cat -A` and `sed -n l`)
`Ctrl+V` then `Ctrl+Arrow` does now yield anything at all. hence when I press `[Ctrl+V][Ctrl+Left][Enter]`, I get `^M` as if I never pressed `Ctrl+left`.
By the way, I just tested `Ctrl+Home` and `Ctrl+End` and it's the same, no reaction.
So all the Ctrl+[Cursor Keys](https://github.com/mintty/mintty/wiki/Keycodes#cursor-keys) do not work properly for me :(
I forgot to mention that `Shift+Cursor Key` and `Alt+Cursor Key` do work correctly.

**Comment by mintty** [comment]
You could post your mintty config file here, for test reference.
However, the fact that it works for you in vim suggests that it's not a mintty issue.

**Comment by Camusensei** [comment]
```
BoldAsFont=no
RightClickAction=paste
CursorType=block
CursorBlinks=no
BellFlash=no
ScrollMod=ctrl
PgUpDnScroll=no
Term=xterm
```
Thank you for your help, I could narrow it down to the ScrollMod. You should be able to reproduce it now.

**Comment by mintty** [comment]
There is a new feature, prompt skip scrolling (#569), which needed two new shortcuts.
They are by default assigned to Shift+cursor-left/right.
With the ScrollMod option you reassign the functions to the respectively modified cursor keys.
I forgot to list the new keys in the ScrollMod section of the manual page. That will be fixed, of course.
Apart from that, the ScrollMod feature, along with other Shortcut features, obviously needs to steal certain key mappings from normal usage, and the new feature takes 2 more, within the logical domain that's already occupied.

**Comment by Camusensei** [comment]
Very clear explanation, thank you.
I don't think there is another solution to this issue than to reassign the `ScrollMod` key back to `Shift` and to get used to scrolling with `Shift` then :)

**Comment by mintty** [comment]
Released 2.7.6.
